db/migrate/20210802074146_create_freaks.rb

```diff
@@ -1,13 +1,25 @@
 # frozen_string_literal: true
 
 class CreateFreaks < ActiveRecord::Migration[6.1]
+  def up
```

Comment:
Don't modify old migrations. All new changes should happen in new migrations.

Reply:
Ok, but this was more ease for me to modify old migration. I don't use that approach in production

Reply:
Being easier doesn't make it correct. Modifying current migrations can lead to issues when rolling back from newer migrations or future migrations from the one that was changed might not work correctly anymore. Avoid rewriting history since it can cause all sorts of issues when it comes to migrations.

Reply:
Ok, I will do a new migration next time. But for now it's ok?

Reply:
The arguments above should give you the answer now and in the future.